routers/web/repo/issue.go

```diff
@@ -136,6 +138,28 @@ func MustAllowPulls(ctx *context.Context) {
 	}
 }
 
+func dependenciesToHTML(ctx *context.Context, dependencies []*issues_model.DependencyInfo) template.HTML {
+	if len(dependencies) == 0 {
+		return ""
+	}
+
+	htmlCode := "<span>"
+
+	for index, dependency := range dependencies {
+		if index != 0 {
+			htmlCode += `<span>,</span>`
+		}
+		anchorClasses := "gt-ml-2"
+		if dependency.Issue.IsClosed {
+			anchorClasses += " gt-line-through"
+		}
+		htmlCode += fmt.Sprintf(`<a href="%s" data-tooltip-content="#%d %s" class="%s">#%d</a>`,
+			dependency.Issue.Link(), dependency.Issue.Index, templates.RenderEmoji(ctx, dependency.Issue.Title), anchorClasses, dependency.Issue.Index)
```

Comment:
I never remember which template functions escape HTML, so please check that this does by creating an issue with title `<script>alert("xss")</script>` please :)

Reply:
![image](https://github.com/go-gitea/gitea/assets/72873130/9914b08b-027d-4c41-8f16-a36582044531)

Reply:
Ok, it escapes, so all fine.